### PR TITLE
VAP-3: updated HTML to be roughly 508 compliant

### PR DESCRIPTION
**Pull request by djdenv**
With an internal system like this, it's tricky to run external tools against it for accessibility. The POST URL also makes it difficult to point it to any of the forms. The accessibility analyzer I ran was one built into my IDE (IntellJ). The changes herein resolve all issues reported by that inspection.

 - Added labels (albeit hidden, which is acceptable) to all form inputs
 - Updated image scaling to use CSS over width property

### CODE REVIEW

**Comment by djdenv** [comment]
Note, the hidden labels are made hidden by use of a custom `aria-hidden` CSS class which you'll see in the vapals.css file.

File: docs/mockups/background.html [hunk 1]
```diff
@@ -76,8 +76,8 @@
       </span>
      </button>
      <a class="text-nowrap" href="home.html" style="text-decoration: none">
-      <img alt="VAPALS-ELCAP Logo" class="hidden-xs" height="50" src="../www/img/logo_transparent.png" title="VAPALS-ELCAP"/>
-      <img alt="VAPALS-ELCAP Logo" height="50" src="../www/img/logo_name.png" title="VAPALS-ELCAP"/>
+      <img alt="VAPALS-ELCAP Logo" class="hidden-xs" src="../www/img/logo_transparent.png" style="height:50px;" title="VAPALS-ELCAP"/>
```
Comment (dlwicksell) [inline docs/mockups/background.html]: Why did you move the height from the tag attribute to CSS? Won't this cause the page to potentially reflow after it processes the CSS? Just curious why you changed it. Was it required for 508 compliance?

File: docs/mockups/background.html [hunk 2]
```diff
@@ -1647,7 +1656,7 @@ <h4 class="modal-title">
     </span>
    </div>
   </footer>
-  <script language="JavaScript">
+  <script lang="JavaScript">
```
Comment (dlwicksell) [inline docs/mockups/background.html]: I thought the lang attribute was for specifying the human language of the content, not the type of script language. I would expect a type="text/javascript" or nothing here. What am I missing?

File: docs/mockups/followup.html [hunk 3]
```diff
@@ -310,10 +319,10 @@ <h3>
      </div>
      <div class="row">
       <div class="col-md-4">
-       <label class="visible-xs visible-sm required" for="siopss-c">
+       <label class="visible-xs visible-sm required" for="">
```
Comment (dlwicksell) [inline docs/mockups/followup.html]: I'm curious why you changed the for attribute to the empty string, rather than just removing it.

File: docs/www/report.html [hunk 4]
```diff
@@ -154,7 +154,7 @@ <h2>
 </pre>
    <div class="pull-right">
     <button class="btn btn-default" data-clipboard-target="#report-text">
-     <i alt="Copy to clipboard" class="glyphicon glyphicon-copy">
+     <i class="glyphicon glyphicon-copy" title="Copy to clipboard">
```
Comment (dlwicksell) [inline docs/www/report.html]: Is title better for 508 compliance than alt? I would have thought the opposite? Why is it better?

File: docs/mockups/background.html (hunk 1 above)
Comment (djdenv) [reply docs/mockups/background.html]: It was not for 508, but was something the IDE complained about, which really just complained that the image size was not the same size as it's rendered size. I will back this out and actually create an image that is the appropriate size, so no scaling will be needed. That will make the page load faster too.

File: docs/mockups/background.html (hunk 2 above)
Comment (djdenv) [reply docs/mockups/background.html]: Not sure about that, but `language` has been deprecated. All browsers (supposedly) use `text/javascript` as the default `type` so that can and is omitted. I think we can remove the newly added `lang` attribute too since it really adds no value.

File: docs/mockups/followup.html (hunk 3 above)
Comment (djdenv) [reply docs/mockups/followup.html]: That wasn't the intent. The Jinja macro always puts in a for attribute even when there is no value for the argument. I'll change that to just omit the attribute when there is no value passed.

File: docs/www/report.html (hunk 4 above)
Comment (djdenv) [reply docs/www/report.html]: There's a lot of funky history with the `<i>` element, but it is not an image as you might think. Technically, it's used for alternate text styling (like italics). It is odd that it's used to render icons, but it is what it is. That said, I don't think the alt attribute would work here on a screen reader. The title attribute does get read with screen readers and also provides that text as a tooltip as an added bonus.

File: docs/mockups/background.html (hunk 1 above)
Comment (dlwicksell) [reply docs/mockups/background.html]: Sounds good.

File: docs/mockups/background.html (hunk 2 above)
Comment (dlwicksell) [reply docs/mockups/background.html]: I knew that language was deprecated. I didn't think you could use lang like that in this tag, but I do remember that having nothing there will default to type="text/javascript," so that sounds good.

File: docs/mockups/followup.html (hunk 3 above)
Comment (dlwicksell) [reply docs/mockups/followup.html]: Makes sense.

File: docs/www/report.html (hunk 4 above)
Comment (dlwicksell) [reply docs/www/report.html]: That's interesting.

File: docs/mockups/background.html [hunk 5]
```diff
@@ -76,8 +76,8 @@
       </span>
      </button>
      <a class="text-nowrap" href="home.html" style="text-decoration: none">
-      <img alt="VAPALS-ELCAP Logo" class="hidden-xs" src="../www/img/logo_transparent.png" style="height:50px;" title="VAPALS-ELCAP"/>
-      <img alt="VAPALS-ELCAP Logo" src="../www/img/logo_name.png" style="height:50px;" title="VAPALS-ELCAP"/>
+      <img alt="VAPALS-ELCAP Logo" class="hidden-xs" src="../www/img/logo_transparent.png" title="VAPALS-ELCAP" width="50"/>
```
Comment (dlwicksell) [inline docs/mockups/background.html]: I assume that you meant to change height to width here, when you changed attributes.

Comment (djdenv) [reply docs/mockups/background.html]: disregard. rebaseing to master is causing some issues. It will merge correctly once I fix that.

Comment (dlwicksell) [reply docs/mockups/background.html]: Okay.

**Comment by djdenv** [comment]
GitHub is being weird showing the merge commit. I'm going to abandon this PR and start a new one that shows the proper files edited.
